discretenet/generator.py

```diff
@@ -4,6 +4,7 @@
 
 from joblib import Parallel, delayed
 import numpy as np
+import random
```

Comment:
Minor, can you move this import up to below the pathlib import (and leave an empty line before the joblib import)? 

Imports are organized by standard library modules, third party modules, and then local modules, each in alphabetical order (there's no flake8 check for this yet, I should have written it down somewhere!)

Reply:
Updated